doc/sidebar.html

```diff
@@ -221,6 +221,12 @@
           </ul>
         </details>
       </li>
+      
+      <li>
+        <a href="LINKROOTindex.html#mlpack-on-embedded-systems">
+          Embedded Applications
+        </a>
```

Comment:
Up to you, you could also make a details section to link to this page.

Reply:
I see your point, but let us wait, let us make a couple of articles first and let us see, it would be easier to find a better location for them, because for now I am not sure of this one, and I am pretty sure that once we have all of them it would be easier to do sections and sub section

Reply:
Sounds good :+1: